### DISCUSSION TITLE
PCI DSS Copmliance

### DISCUSSION THREAD

**Discussion by melmandd** [gravitational/teleport#10612] [Q&A] [answered]
Hi all!
We use Teleport (on premise) to access in to our PCI DSS scope with 2fa.
Where can I find answers to questions about the following requirements of the PSIDSS standard or change them:
...PCI DSS 3.2.1
8.1.6 Limit repeated access attempts by locking out the user ID after not more than six attempts.

8.1.7 Set the lockout duration to a minimum of 30 minutes or until an administrator enables the user ID.

8.1.8 If a session has been idle for more than 15 minutes, require the user to re-authenticate to re-activate the terminal or session.

8.2.3 Passwords/passphrases must meet the following: • Require a minimum length of at least seven characters. • Contain both numeric and alphabetic characters. Alternatively, the passwords/ passphrases must have complexity and strength at least equivalent to the parameters specified above.

8.2.4 Change user passwords/passphrases at least once every 90 days.

Thanks

**Comment by webvictim** [comment, marked as answer]
Teleport does automatically lock users after a number of invalid password attempts, but I don't believe we publish the details of exactly how many attempts anywhere (other than in the code). Teleport also doesn't have the ability to set arbitrary password restrictions on local user accounts.

The general intention is that users should look to implement such restrictions via their SSO provider and then use that to sign into Teleport, rather than using Teleport's native authentication to achieve those restrictions.
